src/components/RichSelect/__stories__/LoadingExample.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { ComponentStory } from '@storybook/react'
+import { useEffect, useState } from 'react'
+import RichSelect from '..'
+import { Button, Loader } from '../..'
+
+// @fixme is this really necessary??
```

Comment:
Can it be removed?

Reply:
This is done in another PR https://github.com/scaleway/scaleway-ui/pull/2098